nixos/modules/hardware/brillo.nix

```diff
@@ -0,0 +1,26 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let
+  cfg = config.hardware.brillo;
+in
+{
+  options = {
+    hardware.brillo = {
+      enable = mkOption {
+        default = false;
+        type = types.bool;
+        description = ''
+          Enable brillo in userspace.
+          This will allow brightness control from users in the video group.
+        '';
+
+      };
+    };
+  };
+
+
+  config = mkIf cfg.enable {
```

Comment:
This might also need `environment.systemPackages` for polkit rules.

Reply:
I have done this and now pkexec works but only if you input the full path (e.g. `pkexec /nix/store/ddavxh7mmbmxds6wpvqws1jw34j3nldh-brillo-1.4.3/bin/brillo -A 5`) I don't know if this is how it should be

Reply:
Should the polkit file be patched so that it lists the executable as something else